app/components/card/story/FLPOStoryCard.vue

```diff
@@ -241,6 +241,9 @@
           axios.get(topoFile)
             .then(response => {
               this.selectedTopology = response.data;
+              if (this.loadingStatusDataset == 'SUCCESS'){
```

Comment:
This will trigger the chart update ONLY if the topoFile is loaded after the dataset. If, for any reason, the dataset takes too long to load (and the topoFile is downloaded faster), you'll have to trigger the chart update another way.